src/modules/claim/services/claim-issuance.service.ts

```diff
@@ -83,10 +85,41 @@ export class ClaimIssuanceService {
       claimType: dto.claimType,
     });
 
-    await this.roleService.verifyEnrolmentPrecondition({
-      claimType: dto.claimType,
-      userDID: dto.requester,
-    });
+    const enrolmentPreconditions =
```

Comment:
It would be easer to read and test if verification of enrollment precoditions was moved in separate method